bitnami/kafka/README.md

```diff
@@ -940,8 +940,9 @@ This new release of the bitnami/kafka container includes a refactor in its logic
   New variables have been added for this purpose:
   * KAFKA_CONTROLLER_USER - Username for the controller communications when SASL is enabled.
   * KAFKA_CONTROLLER_PASSWORD - Password for the controller communications when SASL is enabled.
-* Removed the `"${KAFKA_CONFIG}/server.properties"` vs `${KAFKA_CONFIG}/kraft/server.properties` when using Zookeeper of KRaft mode.
-  By default, Kafka will use `"${KAFKA_CONFIG}/server.properties"`, but the base file has been modified at build time to remove references to both modes.
+* Removed the `${KAFKA_CONFIG}/server.properties` vs `${KAFKA_CONFIG}/kraft/server.properties` when using Zookeeper or KRaft mode.
+  By default, Kafka will use `${KAFKA_CONFIG}/server.properties` which is generated by `${KAFKA_CONFIG}/server.properties.original` based on environment variables.
+  And the base file `${KAFKA_CONFIG}/server.properties.original` has been modified at build time to remove references to both modes.
```

Comment:
@yaohwu this isn't actually correct: "the base file `${KAFKA_CONFIG}/server.properties.original` has been modified at build time to remove references to both modes"

As you can check below, `${KAFKA_CONFIG}/server.properties.original` is the original Kafka configuration file and references to both modes are removed during the container initialization when there's no custom configuration file mounted:

- https://github.com/bitnami/containers/blob/main/bitnami/kafka/3.9/debian-12/rootfs/opt/bitnami/scripts/libkafka.sh#L928-L932
- https://github.com/bitnami/containers/blob/main/bitnami/kafka/3.9/debian-12/rootfs/opt/bitnami/scripts/libkafka.sh#L1037-L1065

Reply:
got it.

> And the base file ${KAFKA_CONFIG}/server.properties.original is the original Kafka configuration file. If no custom configuration file is mounted, references to both modes will be removed during container initialization.

Is this expression appropriate?

Reply:
yes! That sounds better